.github/workflows/prod-ci-cd.yml

```
@@ -0,0 +1,99 @@
+          key: ${{ secrets.PROD_SSH_KEY }}
+          port: ${{ secrets.PROD_PORT }}
+          script: |
+            docker pull ${{ needs.build-and-push.outputs.image_tag }}
+            docker stop wonderwander-prod || true
+            docker rm wonderwander-prod || true
+            docker run -d \
+              --name wonderwander-prod \
+              --restart unless-stopped \
+              -p 80:8080 \
+              -e SPRING_PROFILES_ACTIVE=prod \
+              ${{ needs.build-and-push.outputs.image_tag }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)